tapdb/sqlc/migrations/000007_universe.up.sql

```diff
@@ -81,3 +81,29 @@ CREATE VIEW universe_stats AS
     FROM universe_events u
     JOIN universe_roots roots ON u.universe_root_id = roots.id
     GROUP BY roots.asset_id, roots.group_key, roots.namespace_root;
+
+-- This table contains general configuration for universe federation syncing.
+CREATE TABLE IF NOT EXISTS federation_general_sync_config(
+    -- This ID column is used to ensure that only one row exists in this table.
+    id INTEGER PRIMARY KEY,
+
+    -- This flag indicates that, in general, only issuance proofs should be
+    -- synced.
+    only_sync_issuance_proofs BOOLEAN NOT NULL
+);
+
+-- This table contains universe (asset/asset group) specific federation sync
+-- configuration.
+CREATE TABLE IF NOT EXISTS federation_uni_sync_config (
+    -- This column contains the byte serialized ID of the universe that this
+    -- configuration applies to.
+    universe_id BLOB NOT NULL PRIMARY KEY,
```

Comment:
Re that other TLV, we have the namespace root here, which I think serves the same purpose? https://github.com/lightninglabs/taproot-assets/blob/580c7037f58db3dc91d9c73d1b17e151383b62df/tapdb/sqlc/migrations/000007_universe.up.sql#L5-L9

Then we have a method that takes the ID and maps it to a hex string.